frontend/plan/types.ts

```diff
@@ -177,7 +182,7 @@ export interface Friendship {
 export interface FilterData {
     searchString: string;
     searchType: string;
-    selectedReq: { [K in string]: boolean };
+    selectedAttrs: { [K in string]: boolean };
```

Comment:
Can we make this type more strict by defining what attributes are available?

Reply:
Attributes are dynamic (e.g., we fetch them from the database) so likely not